examples/templating/tera/src/main.rs

```diff
@@ -1,19 +1,22 @@
 //! An example usage of Tera template engine working with Gotham.
 #[macro_use]
 extern crate lazy_static;
-#[macro_use]
-extern crate tera;
 
 use gotham::state::State;
 use tera::{Context, Tera};
 
-// Assuming the Rust file is at the same level as the templates folder
-// we can get a Tera instance that way:
 lazy_static! {
-    pub static ref TERA: Tera =
-        compile_templates!(concat!(env!("CARGO_MANIFEST_DIR"), "/templates/**/*"));
+    pub static ref TERA: Tera = {
+        let tera = match Tera::new(concat!(env!("CARGO_MANIFEST_DIR"), "/TEMPLATES/**/*")) {
+            Ok(t) => t,
+            Err(e) => {
+                println!("Parsing error(s): {}", e);
+                ::std::process::exit(1);
+            }
+        };
```

Comment:
Can't we just use `.expect` instead of the `match` block here?

Reply:
We could but Ideally wouldn't we want to keep in on par with Tera's documentation?

Reply:
I think improving readability is more important than using code from Tera's documentation